typescript/sdk/src/token/EvmERC20WarpModule.ts

```diff
@@ -93,22 +93,22 @@ export class EvmERC20WarpModule extends HyperlaneModule<
     const transactions = [];
 
     transactions.push(
-      ...(await this.updateIsm(actualConfig, expectedConfig)),
-      ...this.updateRemoteRouters(actualConfig, expectedConfig),
-      ...this.updateOwnership(actualConfig, expectedConfig),
+      ...(await this.createIsmUpdateTxs(actualConfig, expectedConfig)),
+      ...this.createRemoteRoutersUpdateTxs(actualConfig, expectedConfig),
+      ...this.createOwnershipUpdateTxs(actualConfig, expectedConfig),
```

Comment:
personally not a fan of these renames but not super important to me given they are not really the top level module API